### PR TITLE
feat(service-logs): new design

### PR DESCRIPTION
**Pull request by RemiBonnet**
# What does this PR do?

https://qovery.atlassian.net/jira/software/projects/FRT/boards/23?selectedIssue=FRT-1289

---

## PR Checklist

- [ ] This PR introduces breaking change(s) and has been labeled as such
- [ ] This PR introduces new store changes
- [x] I made sure the code is type safe (no any)

### CODE REVIEW

**Comment by RemiBonnet** [comment]
[Qovery Preview](https://www.qovery.com)
---

Qovery can create a Preview Environment for this PR.
To trigger its creation, please post a comment with one of the following command.

| Command | Blueprint environment |
|---|---|
| /qovery []() preview cc1de7de-94e5-40f8-8e45-bc8986d9dfec | [storybook](https://console.qovery.com/organization/3d542888-3d2c-474a-b1ad-712556db66da/project/a64f6cc1-4e34-433b-b8fd-5359716fb01b/environment/cc1de7de-94e5-40f8-8e45-bc8986d9dfec/services/general) |
 /qovery []() preview 28c47145-c8e7-4b9d-8d9e-c65c95b48425 | [staging](https://console.qovery.com/organization/3d542888-3d2c-474a-b1ad-712556db66da/project/a64f6cc1-4e34-433b-b8fd-5359716fb01b/environment/28c47145-c8e7-4b9d-8d9e-c65c95b48425/services/general) |
| /qovery []() preview {all\|UUID1,UUID2,...} | To preview multiple environments |

---

This comment has been generated from [Qovery](https://qovery.com) AI :robot:.
Below, a word from its wisdom :
> Write comments that add new information that doesn’t exist in the code

**Comment by fabienfleureau** [comment]
@CodiumAI-Agent /review

**Comment by RemiBonnet** [comment]
@CodiumAI-Agent /review

**Review by ctjhoa — CHANGES_REQUESTED** [review]

File: libs/domains/service-logs/feature/src/lib/list-service-logs/list-service-logs.tsx [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+    serviceId,
+    enabled: service?.serviceType === 'DATABASE' ? service?.mode === 'CONTAINER' : true,
+  })
+
+  const hasMultipleContainers = new Set(logs?.map((i) => i.container_name)).size > 1
+
+  const table = useReactTable({
+    data: logs,
+    columns: hasMultipleContainers
+      ? [
```
[the remark this answers is not in the document]
Comment (RemiBonnet) [reply libs/domains/service-logs/feature/src/lib/list-service-logs/list-service-logs.tsx]: https://github.com/Qovery/console/pull/1661

File: libs/domains/service-logs/feature/src/lib/hooks/use-service-logs/use-service-logs.tsx
[the remark this answers is not in the document]
Comment (RemiBonnet) [reply libs/domains/service-logs/feature/src/lib/hooks/use-service-logs/use-service-logs.tsx:1]: https://github.com/Qovery/console/commit/ee53d18341a500af9aa6cda6a4afd39f7f1f3f7c

File: libs/domains/service-logs/feature/src/lib/hooks/use-service-logs/use-service-logs.tsx [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+      )
+      .sort((a, b) => (a.created_at && b.created_at ? a.created_at - b.created_at : 0))
+  }, [debouncedServiceMessages, debouncedInfraMessages, enabledNginx, showPreviousLogs, now])
+
+  const debouncedLogs = useDebounce(data, DEBOUNCE_TIME)
+  const pausedDataLogs = useMemo(() => debouncedLogs, [pauseLogs])
+
+  return {
+    data: pauseLogs ? pausedDataLogs : debouncedLogs,
+    isLoading: false,
```
[the remark this answers is not in the document]
Comment (RemiBonnet) [reply libs/domains/service-logs/feature/src/lib/hooks/use-service-logs/use-service-logs.tsx]: It's an oversight, I removed it

**Review by ctjhoa — APPROVED** [review]

**Pull request merged**